certified-connectors/DocJuris/apiDefinition.swagger.json

```diff
@@ -548,7 +548,7 @@
     },
     "/v1/subscriptions/turnTagged": {
       "x-ms-notification-content": {
-        "description": "TurnCreatedMessage",
+        "description": "TurnTaggedMessage",
```

Comment:
@joshuaapple descriptions should be a valid English string, please make sure that this description looks like a sentence. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).